translations/readme/README.es.md

```diff
@@ -1,5 +1,12 @@
 # 🟡 GROC &middot; ![Tamaño del repositorio en GitHub][1] ![Tamaño del código][2]
 
+## _Lee esto en [otros idiomas](translations/readme/)._
+
+![🇦🇩 Catalan][3] ·
```

Comment:
Translate the languages list into its own language, but keep this order (as if they were showed in English).